stacks/towerofhanoi.js

```diff
@@ -0,0 +1,53 @@
+const Stack = require("./Stack");
+
+function TowerOfHanoi (discCount) {
+    this.discCount = discCount;
+    this.towerA = new Stack(discCount);
+    this.towerB = new Stack(discCount);
+    this.towerC = new Stack(discCount);
+
+    var noOfDiscs = discCount;
+    while (noOfDiscs-- > 0) {
+        this.towerA.push("Disc");
+    }
+}
+
+TowerOfHanoi.prototype.startMovingDiscs = function () {
+
+    this.solvePuzzle (this.discCount, "A", "C", "B");
```

Comment:
you could have passed three stacks, would have saved you conditions on line 27,30 and 33.